src/Symfony/Component/AssetMapper/ImportMap/PackageVersionProblem.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\ImportMap;
+
+final class PackageVersionProblem
+{
+    public function __construct(
+        public string $packageName,
+        public string $dependencyPackageName,
+        public string $requiredVersionConstraint,
+        public ?string $installedVersion
```

Comment:
Does it make sense to make them `readonly`?

```suggestion
    public function __construct(
        public readonly string $packageName,
        public readonly string $dependencyPackageName,
        public readonly string $requiredVersionConstraint,
        public readonly ?string $installedVersion
```